src/engines/html5/html5.js

```diff
@@ -242,6 +243,10 @@ export default class Html5 extends FakeEventTarget implements IEngine {
       this._mediaSourceAdapter.destroy();
       this._mediaSourceAdapter = null;
     }
+    if (this._droppedFramesWatcher) {
```

Comment:
any reason why you moved it down here? as it is dependent on _mediaSourceAdapter it could trow error if we destroy it after we destroy adapter
Same for reset above

Reply:
It doesn't matter but I have changed it to keep the flow of code meaningful